js/classes/entities/midground/enemies/eel_boss.js

```diff
@@ -114,8 +124,9 @@ class EelBoss {
             }
     
             // if Chad is close enough, bite him
-            if (this.base.chadDistance() < EelBoss.SCALED_SIZE.x) {
+            if (this.base.chadDistance() < EelBoss.ATTACK_DISTANCE) {
                 if (secondsSinceLastAttack > EelBoss.ATTACK_COOLDOWN) {
+                    console.log("initiating attack")
```

Comment:
If you want to keep this in it may be good to make it clear which entity is starting an attack.

Reply:
Thanks for the heads up. I think I'll get rid of it. 

I spent about 2 hours trying to debug the Eel Boss behavior where, in its **first** attack, it waited like 5 seconds before initiating it. In attacks that followed, it would use an appropriate delay. It was just the first that was bugged. The issue still isn't solved, but it's not a huge deal.

Reply:
The snake had a similar problem, but I don't remember what caused it. Maybe they're related? I agree it's a minor issue though.